### DISCUSSION TITLE
Add support for READ UNCOMMITTED in asyncpg dialect.

### DISCUSSION THREAD

**Discussion by benwah** [sqlalchemy/sqlalchemy#9866] [Ideas]
I've added support for READ UNCOMMITTED in asyncpg here: https://github.com/MagicStack/asyncpg/pull/1039

Now it would be nice to also update the asyncpg dialect here:

https://github.com/sqlalchemy/sqlalchemy/blob/main/lib/sqlalchemy/dialects/postgresql/asyncpg.py#L1033-L1040

To support this isolation level.

**Comment by CaselIT** [comment]
Hi,

I'm not sure what's the point. PG does not have a read uncommitted isolation level.
From the docs:
> In PostgreSQL, you can request any of the four standard transaction isolation levels, but internally only three distinct isolation levels are implemented, i.e., PostgreSQL's Read Uncommitted mode behaves like Read Committed. This is because it is the only sensible way to map the standard isolation levels to PostgreSQL's multiversion concurrency control architecture.

https://www.postgresql.org/docs/current/transaction-iso.html

**Reply by benwah** [reply]
Oh.... damn. I didn't realize.
